src/js/visual/configurator.js

```diff
@@ -527,13 +527,18 @@ ripe.Configurator.prototype.changeFrame = async function(frame, options = {}) {
     // configurator according to the current internal state (in data)
     // this operation waits for the proper drawing of the image (takes
     // some time and resources to be completed)
-    await this.update(
-        {},
-        {
-            animate: animate,
-            duration: animate ? duration : 0
-        }
-    );
+    try {
+        await this.update(
+            {},
+            {
+                animate: animate,
+                duration: animate ? duration : 0
+            }
+        );
+    } catch (error) {

```

Comment:
use try/finally instead of re-throw for situations where cleanup is required (in this case the removal of the no-drag and animating classes)